packages/tokens/tokensstudio-generated/tokens.json

```diff
@@ -571,626 +596,2619 @@
             "type": "boxShadow"
           }
-        }
-      }
-    }
-  },
-  "Device/Desktop": {
-    "post": {
-      "sem": {
-        "device": {
-          "sizing": {
-            "interactive": {
-              "height": {
-                "small": {
-                  "1": {
-                    "value": "{post.core.dimension.32}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.32}"
-                  },
-                  "2": {
-                    "value": "{post.core.dimension.28}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.28}"
-                  }
-                },
-                "medium": {
-                  "1": {
-                    "value": "{post.core.dimension.40}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.40}"
-                  }
-                },
-                "large": {
-                  "1": {
-                    "value": "{post.core.dimension.48}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.48}"
-                  }
-                },
-                "extralarge": {
-                  "1": {
-                    "value": "{post.core.dimension.56}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.56}"
-                  },
-                  "2": {
-                    "value": "{post.core.dimension.56}",
-                    "type": "dimension",
-                    "parent": "Device/Desktop",
-                    "description": "",
-                    "oldValue": "{post.core.dimension.56}"
-                  }
-                }
-              }
-            }
+        },
+        "border": {
+          "focus": {
+            "value": {
+              "color": "{post.core.color.sandgrey.100}",
+              "width": "{post.core.dimension.2}",
+              "style": "solid"
+            },
+            "type": "border",
+            "description": "Has to be set arround an element"
+          },
+          "disabled": {
+            "value": {
+              "color": "{post.core.color.sandgrey.060}",
+              "width": "{post.core.dimension.2}",
+              "style": "dashed"
+            },
+            "type": "border",
+            "description": "To be used for everything disabled that has a natural border"
           }
         }
       }
     }
   },
-  "Device/Tablet": {
+  "Mode/Light": {
```

Comment:
Shouldn't this be lower-cased?

Reply:
Doesn't matter, because it will be outputted lower-cased anyway ;)